common/src/main/java/com/microsoft/identity/common/internal/controllers/ExceptionAdapter.java

```diff
@@ -211,6 +221,43 @@ public static void applyCliTelemInfo(@Nullable final CliTelemInfo cliTelemInfo,
         }
     }
 
+    private static IntuneAppProtectionPolicyRequiredException getIntuneAppProtectionPolicyRequiredException(@NonNull final TokenErrorResponse errorResponse) {
```

Comment:
There is an identical-ish piece of code in PRTController. If you're using this one, can we make that one in PRTController invoke this new function of yours?

Reply:
The input parameters for PrtController is different from the msal's.

Reply:
Let me try to get more details to answer this one better.

Reply:
Here, the IntuneAppProtectionPolicyRequiredException is constructed by the client info returned from Server's token response. However, in the PRT flow, the IntuneAppProtectionPolicyRequiredException is constructed by the silent request parameters.